### ISSUE TITLE
Picovoice Console Issue: very limited usage of words in polish

### ISSUE THREAD

**Issue by predond** [Picovoice/rhino#727]
I'm trying to configure a Rhino Speech-To-Intent context but many if not most words are said to has unknown pronunciation. With that on board I can't make model to handle switching lights. Can I bypass it in any way?

![obraz](https://github.com/user-attachments/assets/5a4e56b0-8680-4a43-aa2b-c0f059a1cec1)

**Comment by albho** [comment]
Thanks for reporting. We've added a ticket to add `kinkiet` to our Polish dictionary, and will update this thread when it gets added.

**Comment by predond** [comment]
Good evening,the problem is that not only that words doesn’t work. Most of polish lamp type names have problems with pronunciation.Also if I can ask for a little upgrade of Rhino web context creator. It would be very useful to enable custom words dictionary in Rhino, just like in speech-to-text dictionary, where I can set up any pronunciations on every word I can imagine. With that when I encounter a word that isn’t recognized by engine I could add my custom pronunciation to it and get my application work without concerning you with every word that is incaccessible.The list of the most important specific words is following:- główna lampa- kinkiet- kinkiety- główną lampę- jadalni- salonu- sypialni- kuchni- ledy- tapety- szafki- lustra- prysznicaAs I noticed most of the words are available only in infinitives. Another good idea is to add microphone icon so I could record sample real voice data for the specific words.These are my ideas as an user. I hope you will consider them.Kind regards,Dawid Krzysiak Od: Albert HoWysłano: wtorek, 27 sierpnia 2024 19:24Do: Picovoice/rhinoDW: predond; AuthorTemat: Re: [Picovoice/rhino] Picovoice Console Issue: very limited usage of words in polish (Issue #727) Thanks for reporting. We've added a ticket to add kinkiet to our Polish dictionary, and will update this thread when it gets added.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
